samples/package.json

```diff
@@ -11,8 +11,9 @@
   "author": "Tiago Siebler",
   "license": "MIT",
   "dependencies": {
+    "axios": "^1.12.2",
```

Comment:
```suggestion
```

Is axios needed directly? I thought only the dependencies use it (no direct dependency)